cpp/daal/include/algorithms/k_nearest_neighbors/bf_knn_classification_model.h

```diff
@@ -52,6 +52,25 @@ enum DataUseInModel
     doNotUse = 0, /*!< The input data and labels will not be the component of the trained kNN model */
     doUse    = 1  /*!< The input data and labels will be the component of the trained kNN model */
 };
+/**
+ * <a name="DAAL-ENUM-ALGORITHMS__BF_KNN_CLASSIFICATION__RESULTTOCOMPUTEID"></a>
+ * Available identifiers to specify the result to compute
+ */
+enum ResultToComputeId
```

Comment:
```suggestion
enum ResultToComputeId : unsigned long long
```
What is the reason `0x0....ULL` instead of just numbers?

Reply:
@KulikovNikita, it isn't C++03

Reply:
If I'm not mistaken values that correspond different options in `enum` must be automatically casted to `unsigned long long` since this `enum` is derived from it. Thus there is no additional reasons to mark them with `ULL` suffix .

Reply:
Moreover. There is a cast from `ResultToComputeId` (that is actually just `unsigned long long`) to `DAAL_UINT64`. Due to standard we can not be sure that we can cast such values without data loss since every greater type should have greater or equal size then previous one.